src/remote/util.py

```diff
@@ -64,8 +65,12 @@ def rsync(
     :param info: True if need to add -i flag to rsync
     :param verbose: True if need to add -v flag to rsync
     :param dry_run: True if need to add -n flag to rsync
+    :param delete: True if all files inside destination directory need to be deleted if they were not bringed from source and
```

Comment:
grammatical nit: 
s/they were not bringed from source/they were not found at source/

Reply:
I'll merge it now and fix the docstring in the release commit